src/View/Antlers/Language/Runtime/RuntimeParser.php

```diff
@@ -369,11 +370,11 @@ protected function renderText($text, $data = [])
 
             $this->nodeProcessor->triggerRenderComplete();
         } catch (AntlersException $antlersException) {
-            if (! class_exists(ViewException::class)) {
-                throw $antlersException;
+            if (class_exists(ViewException::class) || class_exists('Spatie\LaravelIgnition\Exceptions\ViewException')) {
```

Comment:
This is a shame, but I guess we need to support the `facade` and `spatie` versions.

Can you extract this to a method and do `if ($this->isIgnitionInstalled())` ?

Reply:
I've made the edit, however I have seen some odd behavior with exceptions like `Statamic\Modifiers\ModifierNotFoundException`.

With recent versions of Statamic requiring Laravel `^8.48.0 || ^9.0`, might be worth refactoring to use `spatie/laravel-ignition` across both versions (and add it to require-dev)?